quiz/apps.py

```diff
@@ -2,4 +2,4 @@
 
 
 class QuizzConfig(AppConfig):
-    name = 'quizz'
```

Comment:
Generally, if you need to use 'and' in your commit message, you might be doing too many things in the same commit. :D You could have split this in 3 commits:
- fix typo 'quizz' to 'quiz'
- detail view navigation
- quiz functionality